framework/core/src/Forum/Controller/LogOutController.php

```diff
@@ -81,29 +91,29 @@ public function handle(Request $request): ResponseInterface
     {
         $session = $request->getAttribute('session');
         $actor = RequestUtil::getActor($request);
+        $base = $this->url->to('forum')->base();
 
-        $url = Arr::get($request->getQueryParams(), 'return', $this->url->to('forum')->base());
+        $rurl = Arr::get($request->getQueryParams(), 'return');
```

Comment:
Please use a sensible variable name like returnUrl here. Short variables will bring confusion in the long run.